sgl-router/src/grpc_client/sglang_scheduler.rs

```diff
@@ -462,8 +465,8 @@ impl SglangSchedulerClient {
             spaces_between_special_tokens: true,
             ignore_eos: false,
             no_stop_trim: false,
-            n: 1,             // Responses API doesn't support n>1
-            constraint: None, // No constraints - tools handled by MCP
+            n: 1,       // Responses API doesn't support n>1
+            constraint, // Structured output constraint from text field
```

Comment:
Can we simply call `Self::extract_structured_output_constraint(request)?;` here instead of creating constraint on the top? the variable is called only once.